serverless.yml

```diff
@@ -97,15 +107,36 @@ functions:
 resources: # CloudFormation template syntax from here on.
   extensions: ${file(./sls-config-${opt:stage}.yml), null}
   Resources:
+
+    ase-findme-image-upload-bucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: ase-findme-image-upload-bucket
+        AccessControl: PublicRead
+
     usersTable:
       Type: AWS::DynamoDB::Table
       Properties:
         TableName: usersTable
         AttributeDefinitions:
-          - AttributeName: userId
+          - AttributeName: user_id
+            AttributeType: S
+        KeySchema:
+          - AttributeName: user_id
+            KeyType: HASH
+        ProvisionedThroughput:
+          ReadCapacityUnits: 1
+          WriteCapacityUnits: 1
+
+    usersImageTable:
```

Comment:
Should we really structure this table by users? I was thinking about having a location-riddles table where the keys are some form of location-riddle Id and the user who posted it is just an attribute. I guess then each location riddle would have 1 (or more) images that are coupled to that riddle and not each user has multiple images in the same row. But im not sure if I understood your approach correctly

Reply:
I think you are right. but i think we might want the table to still be searchable for user_id as well, as we probably want to be able to display a profile with all the riddles. an other way to do so could be to create a composite primary key with user_id just as the search key